reporting/impl/src/main/java/org/jboss/windup/reporting/rules/generation/techreport/GetTechnologiesIdentifiedForBoxAndRowMethod.java

```diff
@@ -0,0 +1,164 @@
+package org.jboss.windup.reporting.rules.generation.techreport;
+
+import freemarker.ext.beans.StringModel;
+import freemarker.template.TemplateModelException;
+import java.util.*;
+import java.util.logging.Logger;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+import org.jboss.windup.config.GraphRewrite;
+import org.jboss.windup.graph.GraphContext;
+import org.jboss.windup.graph.model.ProjectModel;
+import org.jboss.windup.reporting.freemarker.WindupFreeMarkerMethod;
+import org.jboss.windup.reporting.model.TagModel;
+import org.jboss.windup.reporting.model.TechnologyUsageStatisticsModel;
+import org.jboss.windup.reporting.service.TagGraphService;
+import org.jboss.windup.util.ExecutionStatistics;
+
+/**
+ * Gets the list of TechnologyUsageStatisticsModel-s which should be displayed in the box given by the report "coordinates" tags (subsector/box, row).
+ *
+ * <p> Called from a Freemarker template as follows:
+ *
+ * <pre>
+ *      getTechnologiesIdentifiedForBoxAndRow(
+ *          subsector: TagModel,
+ *          row: TagModel,
+ *          projectToCount: ProjectModel
+ *      ): List<{@link org.jboss.windup.reporting.model.TechnologyUsageStatisticsModel}
+ * </pre>
+ *
+ * @author <a href="http://ondra.zizka.cz/">Ondrej Zizka, zizka@seznam.cz</a>
+ */
+public class GetTechnologiesIdentifiedForBoxAndRowMethod implements WindupFreeMarkerMethod
+{
```

Comment:
This is also something to go away if the 2nd implementation is accepted as better (which I recommend).